README.md

```diff
@@ -68,53 +68,62 @@ You would need to install `apt-transport-https` that supports adding an `https`
 
 here are the steps to perform:
 
-```
-apt-get install -y apt-transport-https
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
-add-apt-repository "deb https://raw.githubusercontent.com/nisabek/icinga2-slack-notifications/master/reprepro general main"
-apt-get update
+```bash
+# apt-get install -y apt-transport-https
+# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
```

Comment:
why the "#" in the beginning of the line?..

Reply:
To denote that it is meant to run in root shell/with sudo perms, as opposed to user ($)

I really just wanted to differentiate shell commands from file examples, as both that and templates are in the readme and it was a tad confusing without it

Reply:
yeah, I'm just a bit worried that that's how you comment in a shell file...and who knows how one runs the commands...from a file or as a user...

Reply:
I've switched to the `console` markdown template and made it clear it is a shell prompt.

I've also highlighted the rest of the blocks as `php`, while not being technically correct, it does highlight the text blocks nearly perfectly.